draft-ietf-ivy-network-inventory-yang.md

```diff
@@ -444,6 +478,14 @@ classes, such as platform software, BIOS, bootloader, and software
 patch information, are outside the scope of this document and defined other documents such as
 {{?I-D.wzwb-ivy-network-inventory-software}}.
 
+### Breakout ports {#ports}
+
+The model defines the 'breakout-channels' presence container to indicate whether the port, which contains the transceiver module, can be configured as a breakout port or not.
+
+It is assumed that a port which supports port breakout can be configured either as a trunk port or as a breakout port.
+
+Reporting whether a port, which supports port breakout, is configured as a trunk or as a breakout port, is outside the scope of the base network inventory model: the model providing the mapping between the topology and the inventory models should instead provide sufficient information to identify how the port is configured and, in case of breakout configuration, which breakout channel is associated with which Link Termination Point (LTP), abstracting a device physical interface within the topology model.
```

Comment:
```suggestion
Reporting whether a port, which supports port breakout, is configured as a trunk or as a breakout port, is outside the scope of the base network inventory model. The model providing the mapping between the topology and the inventory models should provide sufficient information to identify how the port is configured and, in case of breakout configuration, which breakout channel is associated with which Link Termination Point (LTP), abstracting a device physical interface within the topology model.
```